image/helpers.go

```diff
@@ -86,7 +86,7 @@ func DownloadSnap(sto Store, name string, revision snap.Revision, opts *Download
 }
 
 // StoreAssertionFetcher creates an asserts.Fetcher for assertions against the given store using dlOpts for authorization, the fetcher will save assertions in the given database and after that also call save for each of them.
-func StoreAssertionFetcher(sto Store, dlOpts *DownloadOptions, db *asserts.Database, save func(asserts.Assertion) error) *asserts.Fetcher {
+func StoreAssertionFetcher(sto Store, dlOpts *DownloadOptions, db *asserts.Database, save func(asserts.Assertion) error) asserts.Fetcher {
```

Comment:
Not really relevant to this branch, but the doc comment says, "... the fetcher will **save** assertions in the given database and after that also call **save** for each of them".

Maybe the param should be called "saved" instead?